### DISCUSSION TITLE
How does the "disallowed_clients" option work?

### DISCUSSION THREAD

**Discussion by fizsef** [AdguardTeam/AdGuardHome#5599] [Q&A]
I don't understand the "disallowed_clients" function. I read the description on https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration , but I still didn't understand what they are for - "allowed_clients" and "disallowed_clients". Tell us on your fingers how they work. Thanks.

**Comment by ainar-g** [comment]
All queries from a client in the *Disallowed clients* list are dropped.  Meaning that they basically get no processing at all, including not appearing in the query log and statistics.

**Comment by fizsef** [comment]
I see. Thank you very much. I added 127.0.0.1 to "disallowed_clients" so that requests from it would not get into statistics and into the log. But I was not sure of the correctness of such an action.
That's why I'm really looking forward to the option so that requests from a specific client (by IP address or MAC) do not get into statistics and into the log.

**Reply by ainar-g** [reply]
I am once again asking you to use the replies in Discussions.

 >  I added 127.0.0.1 to "disallowed_clients" so that requests from it would not get into statistics and into the log.

You have essentially blocked localhost from using AdGuard DNS.  This is probably not what you want.  [As mentioned previously](https://github.com/AdguardTeam/AdGuardHome/issues/5586#issuecomment-1466433768), there isn't currently a way to ignore queries in statistics and query log based on the client data.

**Reply by fizsef** [reply]
> I am once again asking you to use the replies in Discussions.

Извините, я не понял о чем вы. Английский совсем не знаю, часто непонятно после переводчика, какой смысл имеет контекст.

**Reply by ainar-g** [reply]
В Дискуссиях лучше отвечать снизу под ответом, как вы сделали сейчас, а не писать новый ответ.  Смысл Дискуссий в том, чтобы на первом уровне темы были только ответы, один из которых вы можете признать верным (кнопка *Mark as answer).*  Иначе остальным сложно потом искать и обсуждать ответы.
